zirgen/Conversions/Typing/ZhlComponent.cpp

```diff
@@ -1243,9 +1275,13 @@ void LoweringImpl::gen(SwitchOp sw, ComponentBuilder& cb) {
         addArgumentAliases(sublayout, counter);
       }
       break;
-    case LayoutKind::Mux: {
+    case LayoutKind::MajorMux:
```

Comment:
were the braces around the case block here removed intentionally?

Reply:
Yes, I generally only use braces in switches when there is variable declarations inside the case body.